dockerfiles/Dockerfile-snapp-test-transaction

```diff
@@ -0,0 +1,38 @@
+ARG deb_codename=stretch
+FROM debian:${deb_codename}-slim
+
+# Run with `docker build --build-arg deb_version=<version>`
+ARG deb_version
+ARG deb_codename=stretch
+ARG deb_release=alpha
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN echo "Building image with version $deb_codename $deb_release $deb_version"
+
+COPY scripts/archive-entrypoint.sh /entrypoint.sh
```

Comment:
Remove this line and the next one, i doubt this entrypoint is used anywhere or usable for this purpose